README.md

```diff
@@ -1,5 +1,53 @@
-# Heroku Cloud Native PHP Buildpack
+# Heroku Cloud Native Buildpack: PHP
 
-[![CI](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml)
+[![Cloud Native Buildpacks Registry: heroku/php][registry-badge]][registry-url]
+[![CI on Github Actions: heroku/php][ci-badge]][ci-url]
 
-Heroku's official [Cloud Native Buildpack](https://buildpacks.io) for the PHP ecosystem.
+![Heroku Cloud Native Buildpack: heroku/php][cnb-banner]
+
+`heroku/php` is the [Heroku Cloud Native Buildpack](heroku-buildpacks)
+for PHP applications. It builds PHP application source code into application images with
+minimal configuration.
+
+> [!IMPORTANT]
+> This is a [Cloud Native Buildpack](cnb), and is a component of the [Heroku Cloud Native Buildpacks](heroku-buildpacks) project. If you are instead looking for the Heroku Classic Buildpack for PHP (for use on the Heroku platform), you may find it [here](classic-buildpack).
```

Comment:
Do we need to mention that these are still experimental (or beta)? Similar to:
https://github.com/heroku/cnb-builder-images/blob/e07ddd37f665f4151e9846891caa076a3b9d35a1/README.md?plain=1#L5-L6

Reply:
I'd rather say "beta" than "experimental" given we're presenting this to a large community. I'll update it.